### PR TITLE
[PRM-213]: added routing to penaltes appeals plus penaltyId to model

### PR DESCRIPTION
**Pull request by BackRowOla**

### CODE REVIEW

**Comment by BackRowOla** [comment]
test this please

File: app/controllers/IndexController.scala [hunk 1]
```diff
@@ -57,4 +56,9 @@ class IndexController @Inject()(view: IndexView,
         isAnyUnpaidLSPAndNotSubmittedReturn))
     }
   }
+
+  def redirectToAppeals(penaltyId: String): Action[AnyContent] = authorise.async { implicit request =>
+    logger.debug(s"[IndexController][appealPenalty] redirect to appeals frontend with id $penaltyId")
```
Comment (LewisMitchell) [inline app/controllers/IndexController.scala:61]: The [appealPenalty] should be [redirectToAppeals]

File: app/viewmodels/SummaryCard.scala [hunk 2]
```diff
@@ -23,9 +23,10 @@ case class SummaryCard(
                         cardRows: Seq[SummaryListRow],
                         status: Tag,
                         penaltyPoint: String,
+                        penaltyId: String,
                         footerUrl: String = "#",
```
Comment (LewisMitchell) [inline app/viewmodels/SummaryCard.scala:27]: I think we can remove the footer url now

File: test/views/IndexViewSpec.scala [hunk 3]
```diff
@@ -182,7 +189,7 @@ class IndexViewSpec extends SpecBase with ViewBehaviours {
           doc.select(Selectors.rowItem(4)).text shouldBe pointExpiration
           doc.select(Selectors.summaryCardFooterLink).text shouldBe appealLinkText
           //TODO: change this when we appeal penalties link
```
Comment (LewisMitchell) [inline test/views/IndexViewSpec.scala:191]: we can remove the //TODO now

File: test/controllers/IndexControllerSpec.scala [hunk 4]
```diff
@@ -88,5 +88,14 @@ class IndexControllerSpec extends SpecBase with MockitoSugar {
         }
       }
     }
+
+    "redirectToAppeals" when {
+      "the user wants to appeal a penalty" in {
+        val result = Controller.redirectToAppeals("123456789")(fakeRequest)
+
+        status(result) shouldBe SEE_OTHER
```
Comment (LewisMitchell) [inline test/controllers/IndexControllerSpec.scala:96]: Could we add a check for the redirect location through `redirectLocation(result)`?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
LGTM. Just a few changes

**Review by LewisMitchell — APPROVED** [review]
LGTM. Just running the AT's.

**Pull request merged**